core/evancore.vue.libs/src/components/login/login.ts

```diff
@@ -93,33 +95,33 @@ export default class LoginComponent extends mixins(EvanComponent) {
         this.form.password.valid = await dappBrowser.bccHelper.isAccountPasswordValid(
           bcc,
           this.accountId,
-          this.form.password.value
+          this.form.password.value,
         );
       } catch (ex) {
         this.form.password.value = false;
       }
 
-      // if the password is correct, create the correct active vault in dapp-browser, so other
-      // applications can access it
+      /* if the password is correct, create the correct active vault in dapp-browser, so other
```

Comment:
I'm actually really not happy with this multiline comment style. Also breaks the automatic comment/uncomment functionality in vscode. Would it be possible too fix this in eslintrc?

Reply:
Ofc it's possible to disable that rule. I don't mind disabling it.

Reply:
is disabled ;)